.yamato/ruamel/build.py

```diff
@@ -87,8 +93,7 @@ def get_metafile(metafile_name, unfold_agents_root_keys=[], unfold_test_platform
     yml_dump_files(create_projectcontext_ymls(package_metafile))
 
     # create abv
-    print(f'Running: ABV')
```

Comment:
I know the print statement got removed on master (my mistake when moving the smoke tests to a specific project file), but I think it is actually intended to stay